### ISSUE TITLE
namespace_id registry

### ISSUE THREAD

**Issue by frisitano** [celestiaorg/celestia-app#912]
### Summary

It may be of value to provide a centralised repo where projects / teams can browse and "claim" a namespace_id.  This will allow for quick discovery of namespace_id associated with a project and also potentially prevent overlap.  This is somewhat analogous to the chainID  in Ethereum based chains, although the security implications are different. 

### Problem Definition

May be hard to determine which project is associated with a namespace_id.  Additionally there may be potential that two projects try to use the same namespace_id.

### Proposal

Create a centralised repository where teams can browse and "claim" a namespace_id.

Examples: 

EVM - ChainID:
https://github.com/DefiLlama/chainlist/blob/main/constants/chainIds.js

Substrate ss58 registry:
https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json

### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by rootulp** [comment]
This is a cool idea, thanks for opening an issue @frisitano ! On first glance, I think a repo with such a table would be interesting to browse but the mapping would be voluntary and unenforced (i.e. the repo has no authority to enforce that only rollup A is the sole publisher to namespace id A).

**Comment by frisitano** [comment]
Agreed, the intention of the repo would just be for convenience but as you say will have no authority, similar to how the examples linked above have no authority.
